pylint/checkers/spelling.py

```diff
@@ -229,11 +231,12 @@ class SpellingChecker(BaseTokenChecker):
         (
             "spelling-dict",
             {
-                "default": "",
+                "default": [],
                 "type": "choice",
                 "metavar": "<dict name>",
                 "choices": _get_enchant_dict_choices(enchant_dicts),
```

Comment:
```suggestion
```
We can also remove ``_get_enchant_dict_choices`` if it's not used elsewhere. Disregard this suggestion if the expected use is: ``--spelling-dict=en_gb --spelling-dict=en_us``, but then how would it work in the configuration file ? (It needs to be compatible with command line interface and configuration file at the same time.)

Reply:
I think CSV-representation is reasonable (and backwards-compatible) in the rc-configuration file, array configuration would be better in the TOML file, and repeated parameters (`--spelling-dict=en_gb --spelling-dict=en_us`) (with choice prompts available) are good on the command-line.

That could require refactoring work - I'd be OK looking at developing that as a dependency for this change.

It's possible that the complexity of the results wouldn't be worthwhile, though.  Sometimes simple (even with known limitations) is good.

Reply:
I think it's best to keep it simple and stupid and have a single field, the same one that pyenchant expect to get.